### ISSUE TITLE
About ImageNet-21K Pretrain

### ISSUE THREAD

**Issue by SJLeo** [microsoft/Swin-Transformer#93]
The base tagging of the original imagenet21K is single label. I wonder how to get multi-label information for each image in ImageNet22K.

**Comment by ancientmooner** [comment]
We use single-label for pre-training. 

I have read a paper which converts the original ImageNet21K labels to multiple-label ones, but cannot remember the specific title. I would greatly appreciate if somebody could share the paper under this question.

**Comment by fffffgggg54** [comment]
> I have read a paper which converts the original ImageNet21K labels to multiple-label ones, but cannot remember the specific title. I would greatly appreciate if somebody could share the paper under this question.

I believe this is the paper: [ImageNet-21K Pretraining for the Masses](https://arxiv.org/abs/2104.10972).
